### PR TITLE
Create PRODUCT.md

### PR DESCRIPTION
**Pull request by swighe**

### CODE REVIEW

File: notes/PRODUCT.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Product Thinking
+The art to defining and solving a problem with a specified target audience, strategy, and goal in mind.
```
Comment (slaude) [inline notes/PRODUCT.md]: "art of"?

File: notes/PRODUCT.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+## Layout
```
Comment (slaude) [inline notes/PRODUCT.md]: can you explain what this Layout does; flesh out a more descriptive title for the section

File: notes/PRODUCT.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+The how will we do this. #1 rule : Solve the problem first
+
+### Output
+#### Goals Setting
+What do we plan to achieve? What goals are we setting?
+#### Features
+What features will this manifest as? What will we do to reach our goals?
+
+
+## Links to resources
```
Comment (slaude) [inline notes/PRODUCT.md:35]: if these are articles, make sure these are formatted `[_Title_ - Publication](url)`

File: notes/PRODUCT.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+The art to defining and solving a problem with a specified target audience, strategy, and goal in mind.
+
+## Layout
+In order to ______ (vision) <br>
+our product will solve ________ (target audience) <br>
+problem of _________ (user problem) <br>
+by giving them _________ (strategy).  <br>
+We know if our product works when we see a _________ (goal). <br>
+
+### User First
```
Comment (slaude) [inline notes/PRODUCT.md]: I think these`###`-level sections demand some kind of introductory. Also these are lifted directly from the articles you posted below, so attribution would be important, e.g.

`This section adpated from [Article title](url)`

**Review by slaude — CHANGES_REQUESTED** [review]

File: notes/PRODUCT.md (hunk 1 above)
Comment (swighe) [reply notes/PRODUCT.md]: Fixed

File: notes/PRODUCT.md (hunk 2 above)
Comment (swighe) [reply notes/PRODUCT.md]: Updated

File: notes/PRODUCT.md (hunk 4 above)
Comment (swighe) [reply notes/PRODUCT.md]: Did some searching, I found the following verbiage:
This section adapted from Reference 8.

File: notes/PRODUCT.md (hunk 3 above)
Comment (swighe) [reply notes/PRODUCT.md:35]: Addressed

**Review by slaude — APPROVED** [review]

**Comment by slaude** [comment]
LGTM, thanks @swighe !

**Pull request merged**